app/views/layouts/application.html.erb

```diff
@@ -1,21 +1,38 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>TaskTrackerItis</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<head>
+  <title>TaskTrackerItis</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+</head>
 
-  <body class="center">
-    <% flash.each do |type, msg| %>
-      <div class="flash-<%= type %>">
-        <p><%= msg %></p>
-      </div>
+<body class="center">
+<% flash.each do |type, msg| %>
+  <div class="flash-<%= type %>">
+    <p><%= msg %></p>
+  </div>
+<% end %>
+
+<div class="user-navigation">
+  <% if current_user.nil? %>
+    <%= button_to "Log in", new_sessions_path, method: :get%>
+    <%= button_to "Sign up", new_user_path, method: :get%>
+  <% else %>
+    <% if current_user.first_name.present? || current_user.last_name.present? %>
+      <%= "#{current_user.first_name} #{current_user.last_name}" %>
+    <% else %>
+      <%= current_user.email %>
     <% end %>
-    <%= yield %>
-  </body>
+    |
+    <%= button_to "Log out", sessions_path, method: :delete%>
```

Comment:
```suggestion
    <%= button_to "Log out", logout_path, method: :delete%>
```
You provide route realization of `sessions#destroy` like logout. It will be correct if you change path